packages/core/core/src/PromiseQueue.js

```diff
@@ -0,0 +1,78 @@
+// @flow
+
+type PromiseQueueOpts = {
+  maxConcurrent?: number
+};
+
+export default class PromiseQueue {
```

Comment:
Looks like we also already have a PromiseQueue from parcel 1 in the utils package... Wonder if we can combine them?